meeting-notes/2018/2018-08-27--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,138 @@
+- Done:
+    - Fixed issue with `files.add --cid-version=1` [#1518](https://github.com/ipfs/js-ipfs/issues/1518)
+    - Released js-ipfs 0.31.7 with fix for start failure when preloading is disabled [#1514](https://github.com/ipfs/js-ipfs/issues/1514)
+    - Found and sent a PR to js-ipld to address a bottleneck for IPFS startup time [#145](https://github.com/ipld/js-ipld/pull/145)
+    - Sent PR to fix issue with IPLD resolving a non-existent path - was throwing and not calling callback [#146](https://github.com/ipld/js-ipld/pull/146)
+    - Upgraded the [IPLD explorer CLI tool](https://www.npmjs.com/package/ipld-explorer-cli) to use IPLD directly and resolve all sorts of node types. Also now works with js-ipfs daemon \o/!!!!
+- Blocked:
+    - `null`
+- Next:
+    - get IPNS PR into mergable state [#1400](https://github.com/ipfs/js-ipfs/pull/1400)
+    - base32 CID [#1441](https://github.com/ipfs/js-ipfs/pull/1441)
+    
+@jacobheun
+  - Done:
+    - Peer/Content routing work is pretty much done, just need to add an example to work out any issues, [libp2p#242](https://github.com/libp2p/js-libp2p/pull/242). Allows for multiple routing modules and prioritizes the dht if it exists.
+    - Released a security fix for [js-ipns0.1.3](https://github.com/ipfs/js-ipns/commit/33684e356f1f2fdcd99b2fb85fcc5d52223769a0). Public key validation was not happening properly. npm security advisory was submitted.
+    - DHT update to fix bad char encoding logs [libp2p-kad-dht#33](https://github.com/libp2p/js-libp2p-kad-dht/pull/33)
+    - Delegate content routing fix for complying to dht interface [libp2p-delegated-content-routing#4](https://github.com/libp2p/js-libp2p-delegated-content-routing/pull/4)
+    - Update language for libp2p custom bundle pr for ipfs [ipfs#1470](https://github.com/ipfs/js-ipfs/pull/1470)
+  - Blocked:
+    - Delegate routing. These aren't currently hard blockers but will need to be finished in the next couple weeks.
+      - [libp2p-delegated-content-routing#4](https://github.com/libp2p/js-libp2p-delegated-content-routing/pull/4) dht interface compliance
+      - [go-ipfs#4595](https://github.com/ipfs/go-ipfs/pull/4595) Go delegate endpoints
+      - [infrastructure#426](https://github.com/ipfs/infrastructure/issues/426) Go deployment for delegate routes. Blocked by [go-ipfs#4595](https://github.com/ipfs/go-ipfs/pull/4595)
+  - Next:
+    - Delegate routing example
+    - State machine updates
+    - ipfs-repo configuration updates for better go interop
+
+@vasco-santos
+  - Done:
+    N/A
+  - Blocked:
+    - Several IPNS related PRs
+      - [js-ipfs#1400](https://github.com/ipfs/js-ipfs/pull/1400) - needs final review from @alanshaw, and then being merged and released
+      - [interop#29](https://github.com/ipfs/interop/pull/29) - needs review
+      - [interop#26](https://github.com/ipfs/interop/pull/26) - needs [js-ipfs#1400](https://github.com/ipfs/js-ipfs/pull/1400) merged and then PR review
```

Comment:
@achingbrain @alanshaw @hugomrdias can you review these PRs for @vasco-santos ?

Reply:
Done!